src/routes/work-session/work-session.service.ts

```diff
@@ -84,22 +89,12 @@ export class WorkSessionService {
     });
   }
 
-  async remove(id: number): Promise<void> {
-    await this.findOne(id);
-
-    await this.prisma.workSession.delete({
-      where: { projectId: id },
-    });
-
-    return;
-  }
-
   async join(projectId: number, user: UserDto): Promise<JoinRoomResult> {
     const project = await this.prisma.project.findFirst({
       where: { id: projectId },
     });
     if (project === null) {
-      throw new NotFoundException(`Project with ID ${projectId} not found`);
+      throw new NotFoundException(`Project with ID ${projectId}`);
```

Comment:
Why did you remove "`not found`" as part of the message? Even though it is indeed part of the exception name, it is always much more practical to just show the error message rather than trying to format with the type of error.